modules/subscription/locals.tf

```diff
@@ -85,10 +85,14 @@ locals {
       principal_id = data.azuread_group.dts_operations.object_id
       scope        = "/subscriptions/${azurerm_subscription.this.subscription_id}"
     }
-    #Application Gateway Backend Health Reader
-    #"159469f4-6e46-4b09-aa73-8e2f243aa784" = {
-    #principal_id = azuread_group.groups["Reader"].id
-    #scope        = "/subscriptions/${azurerm_subscription.this.subscription_id}"
-    #}
+
+  }
+  custom_role_assignments = {
+    "Application Gateway Backend Health Reader" = {
+      principal_id = azuread_group.groups["Reader"].id
+      role_id      = "159469f4-6e46-4b09-aa73-8e2f243aa784"
```

Comment:
I don't think this will work across tenants, doesn't this need to come from a variable?